pkg/inttest/http.go

```diff
@@ -109,15 +110,15 @@ func (hc *HTTPClient) Do(t *testing.T, method, path string, requestBody io.Reade
 	return body
 }
 
-// do delegates the request to the underlying HTTP client.
+// do delegate the request to the underlying HTTP client.
 func (hc *HTTPClient) do(t *testing.T, req *http.Request) *http.Response {
 	resp, err := hc.Client.Do(req)
 	require.NoError(t, err, httpClientErrMessage(req.Method, req.URL.Path)+": HTTP request failed")
 	return resp
 }
 
 // GetJSON sends an HTTP GET request to given path. Optional headers are applied to the request. The
-// response body is unmarshaled as JSON into given responseBody. Failure to read or close the HTTP
+// response body is unmarshalled as JSON into given responseBody. Failure to read or close the HTTP
```

Comment:
https://pkg.go.dev/encoding/json uses `marshaled`

Seems like you are changing it to British English
https://www.oxfordlearnersdictionaries.com/definition/english/marshal_2?q=marshaled
I would suggest we stick to US English but am ok not to, at least we should stick to one of the two 😅